packages/engine-server/src/EngineEvents.ts

```diff
@@ -0,0 +1,11 @@
+import { Disposable, Event, NoteChangeEntry } from "@dendronhq/common-all";
+
+/**
+ * Interface for events signaling changes that have been made to engine state
+ */
+export interface EngineEvents extends Disposable {
```

Comment:
since this isn't events but an emitter for it, can we rename to `EngineEventsEmitter`?

Reply:
good call